madhumitha/student_marks.js

```diff
@@ -0,0 +1,49 @@
+function add() {
+    // get input values

```

Comment:
I'm able to add empty rows to the table. Please display some message or disable the button when no values are present

Reply:
Sorry.. I have fixed this and updated the code